docs/api/command-line.md

```diff
@@ -0,0 +1,50 @@
+## Class: CommandLine
+
+> Manipulate the command line arguments for your app that Chromium reads
+
+Process: [Main](../glossary.md#main-process)
```

Comment:
Also link to https://electronjs.org/docs/api/chrome-command-line-switches somewhere in this block?